src/frontends/onnx/frontend/src/op/col2im.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/col2im.hpp"
+
+#include "core/operator_set.hpp"
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace ai_onnx {
+namespace opset_1 {
+ov::OutputVector col2im(const ov::frontend::onnx::Node& node) {
+    const auto data = node.get_ov_inputs().at(0);
+    const auto output_shape = node.get_ov_inputs().at(1);
+    
+    std::vector<int64_t> kernel_shape;
+    if (node.has_attribute("kernel_shape")) {
+        kernel_shape = node.get_attribute_value<std::vector<int64_t>>("kernel_shape");
+    } else {
+        FRONT_END_GENERAL_CHECK(false, "Col2Im requires 'kernel_shape' attribute.");
+    }
+    const auto dilations = node.get_attribute_value<std::vector<int64_t>>("dilations", {1, 1});
+    const auto pads = node.get_attribute_value<std::vector<int64_t>>("pads", {0, 0, 0, 0});
+    const auto strides = node.get_attribute_value<std::vector<int64_t>>("strides", {1, 1});
+    
+    return std::make_shared<v10::Col2Im>(
```

Comment:
The Col2Im is in `v15` namespace:
https://github.com/openvinotoolkit/openvino/blob/f347a9612e48299f1eace847bd27b02359e99437/src/core/include/openvino/op/col2im.hpp#L11-L14 
```suggestion
    return std::make_shared<v15::Col2Im>(
```